sdk/core/core-rest-pipeline/package.json

```diff
@@ -124,6 +124,7 @@
     "mocha-junit-reporter": "^2.0.0",
     "mocha": "^7.1.1",
     "prettier": "^2.5.1",
+    "puppeteer": "^13.5.1",
```

Comment:
I do not see this being used anywhere in this PR? was it missing or is it being added by accident?

Reply:
I was trying to run the browser tests to make sure my change was fine and ran into this.
![image](https://user-images.githubusercontent.com/10452642/157790614-fbfcaf7b-53a1-4b09-b453-8b7d82cd9d91.png)

Looked deeper and realized puppeteer library which is used in karma.conf to get the browser is not part of the devDependencies. Hence added it.

Reply:
However, it may have worked everywhere else for anyone else simply because rush mixes all the devDependencies and doesn't complain of missing libraries in package.json.

Reply:
Thanks for clarifying, that change is fine then.